README.md

```diff
@@ -10,6 +10,110 @@ Centos 6
 
 Centos 7?
 
+## Usage
+
+### Resource
+
+```
+    python_webapp "python_app_name" do
+      option "option_value"
+    end
+```
+
```

Comment:
Please add an extra newline here to create an actual newline between the example resource and the table of options.